thunx-api-spring-boot-starter/src/main/resources/spring.provides

```diff
@@ -0,0 +1 @@
+provides: thunx-spring
```

Comment:
Spring Boot itself removed all spring.provides files in 2018
See https://github.com/spring-projects/spring-boot/issues/12435 for more information